pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -202,6 +202,14 @@ stdenv.mkDerivation {
         basename=$(basename "$variant")
         wrap $basename ${./ld-wrapper.sh} $variant
       done
+    ''
+    # Copy the lld-* variants if available.
+    + lib.optionalString isLLVM ''
```

Comment:
This check isn't really necessary — non-LLVM simply won't have any lld-* things.

Reply:
Right, isLLVM avoids a mass rebuild though, if we remove it, this has to go to staging.

Reply:
Better quality code forever >>> avoiding a mass rebuild once.  If you want, you could put the mass-rebuild-avoiding version in master, and then clean it up in staging.

Reply:
Right, agreed.

Reply:
Moved away, I will re-introduce this when everything around this piece of code makes sense as you rightly mentioned it.